src/pybind/mgr/rook/ci/scripts/bootstrap-rook-cluster.sh

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env bash
+
+set -eEx
+
+: ${CEPH_DEV_FOLDER:=${PWD}}
+KUBECTL="minikube kubectl --"
+
+# We build a local ceph image that contains the latest code
+# plus changes from the PR. This image will be used by the docker
+# running inside the minikube to start the different ceph pods
+LOCAL_CEPH_IMG="local/ceph"
+
+on_error() {
+    echo "on error"
+    minikube delete
+}
```

Comment:
the `cleanup` should happen on error too right?

Reply:
@nizamial09 the `cleanup` normally is to remove anything that has been generated during the bootstrap but it's not longer needed.